### ISSUE TITLE
Global Alliance Variant schema design decisions

### ISSUE THREAD

**Issue by larrybabb** [tnavatar/clingen-data-model#61]
Here are some very interesting high level discussions on high level principles regarding variant representation that have a context in the fundamental principles of our canonicalization concept.

From Bob Millius at the National Marrow Donor Program

I showed the clingen model to some of our developers, and they asked me to share these links from the Global Alliance to the the HL7 CG group. 

https://github.com/ga4gh/schemas

Much of the publicly accessible discussion is on github issues

basic standardisation - deletion alleles and start/stop coordinates
https://github.com/ga4gh/schemas/issues/168

HGVS nomenclature benchmarking
https://github.com/ga4gh/schemas/issues/215

Best practices for joining NGS-derived & clinical variation databases
https://github.com/ga4gh/schemas/issues/159

but the real meat is in the schema itself

https://github.com/ga4gh/schemas/blob/master/src/main/resources/avro/variants.avdl

Bob

— 
Bob Milius, Ph.D.
Senior Data Analyst, Bioinformatics Research
National Marrow Donor Program
3001 Broadway Street N. E. Suite 100, Minneapolis, MN 55413-1753

Office: 612-627-5844 | Cell: 612-810-2892

**Comment by cbizon** [comment]
Larry - other than the 0/1 discussion (and the relative size of the groups!) , is there a particular message that you would take from these discussions?

**Comment by larrybabb** [comment]
I did a cursory read through the the last issue on ["Best practices for joining NGS-derived & clinical variation databases"](https://github.com/ga4gh/schemas/issues/159) and I came away thinking that we need to be involved with this group and these discussions.  They seem to be moving very fast and have some great committed resources with tons of expertise.  (but that's just an impression)

They do reference the idea of canonicalization along the way and seem somewhat dismissive of it, like it isn't available or can't be done realistically.  Without a deep dive into the thread it is difficult to get a strong sense of the scope.
